tests/sampling/test_forward.py

```diff
@@ -1669,6 +1670,20 @@ def test_Triangular(
         assert prior["target"].shape == (prior_samples, *shape)
 
 
+def test_get_constant_coords():
+    with pm.Model() as model:
```

Comment:
Can we add a more integration-like test like?

Spit-balling, perhaps something like:

```python
with pm.Model() as m:
  m.add_coord("trial", length=3)
  x = pm.Normal("x", shape="trial")
  y = pm.Deterministic("y", x.mean())

# pass dims as well  
idata = az.from_dict("x": [[np.pi, np.pi, np.pi]])

with m:
  pp1 = pm.sample_posterior_predictive(idata, var_names=["y"]).posterior_predictive

assert pp1["y"] == np.pi

with m:
  # change coord length 
  pp2 = pm.sample_posterior_predictive(idata, var_names=["y"]).posterior_predictive

assert pp2["y"] != np.pi

```

Reply:
Does this look ok? https://github.com/pymc-devs/pymc/pull/7381/commits/d40fffec21077e7b4906b320f0605c5f4e67ddcb